stories/12-Accordion.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {Fragment} from "react";
+import {storiesOf} from "@storybook/react";
+import Accordion from "../src/accordion/Accordion";
+
+storiesOf("Accordion", module).add("Accordion", () => (
+  <Fragment>
+    <p>
+      {
+        "Accordion with defaultExpanded={[\x221\x22]} allowZeroExpanded={false} allowMultipleExpanded={false}"
+      }
+    </p>
+    <Accordion defaultExpanded={["1"]}>
+      <Accordion.Item accordionId={"1"} header={"Accordion header 1"}>
+        Content of the Accordion 1
+      </Accordion.Item>
+      <Accordion.Item accordionId={"2"} header={"Accordion header 2"}>
+        Content of the Accordion 2
+      </Accordion.Item>
```

Comment:
Let's left an empty line between sibling components and use curly braces and double quotes for strings like `{"some string"}` 

```suggestion
      <Accordion.Item accordionId={"1"} header={"Accordion header 1"}>
        {"Content of the Accordion 1"}
      </Accordion.Item>
      
      <Accordion.Item accordionId={"2"} header={"Accordion header 2"}>
        {"Content of the Accordion 2"}
      </Accordion.Item>
```